lib/sass_spec/test.rb

```diff
@@ -138,6 +67,34 @@ def run_spec_test(test_case, options = {})
   end
 end
 
+
+def normalize_error_output(error, options)
+  error_message = ""
+  consume_next_line = false
+  error.each_line do |line|
+    if consume_next_line
+      next if line.strip == ""
+      error_message += line
+      break
+    end
+    if (line =~ /DEPRECATION WARNING/)
+      if line.rstrip.end_with?(":")
+        error_message = line.rstrip + "\n"
+        consume_next_line = true
```

Comment:
This lets us verify deprecation messages match up better than just the first line can accomplish.